test/rules/always_put_control_body_on_new_line.dart

```diff
@@ -19,6 +19,13 @@ testIfElse() {
     return; // OK
   else return; // LINT
 
+  if (false) {
+  }
+  else if (false) { // OK
```

Comment:
It probably doesn't matter for this lint, but I think the formatter would format this as
```
if (false) {
} else if (false) {
} else {
}
```
We might want to make sure that formatted code is also handled correctly.